.github/actions/build-dotnet/Build-DotNetSolutionOrProject.ps1

```diff
@@ -51,7 +51,7 @@ if (Test-Path src/Utilities/Lombiq.Gulp.Extensions/Lombiq.Gulp.Extensions.csproj
 # This prepares the solution or project with the Lombiq.Analyzers files. The output and exit code are discarded because
 # they will be in error if there is a project without the LombiqNetAnalyzers target. Then there is nothing to do, and
 # the target will still run on the projects that have it.
-dotnet msbuild '-target:Restore;LombiqNetAnalyzers' $SolutionOrProject | Out-Null || bash -c 'true'
+dotnet msbuild '-target:Restore;LombiqNetAnalyzers' $SolutionOrProject | Out-Null -or bash -c 'true'
```

Comment:
Also point all the affected action calls in `build-and-test-orchard-core` to the issue branch so these script changes are tested.

Reply:
Revisiting this now, what's the expected output here? Both of those (one the two sides of the `|`) commands return a boolean value and they are just ORed together?

Btw Windows PowerShell doesn't accept this and I'm still getting analyzer errors locally, but the code analysis passes in GHA. I'll undo the changes that have some uncertainty in them or has a potential for side effects and see what the code analysis says in GHA.

Reply:
The double-pipe is not an OR operator: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_pipeline_chain_operators?view=powershell-7.3#long-description However, I'm not clear on why we need this bash trick. @DAud-IcI you've written this originally in 89118d3b8e11af2cc5216f532b349f20cddf64f9. Is this to not fail the script if the `dotnet msbuild` command fails ("exit code are discarded")?

Reply:
I've reverted this change, since LGHA doesn't run analysis on Windows PS, only PS Core, so `||` is allowed.